primitives/manta/src/assets.rs

```diff
@@ -157,9 +136,29 @@ where
     >;
 }
 
+/// Bound for SBT Metadata Size (Could make generic over runtimes, but simpler to just hardcode)
+pub type SbtBound = ConstU32<200>;
```

Comment:
Another reason to make this generic is to be less coupled with primitives.
Up to you though, or we can just log it for now.

Reply:
Is generic now